tests/resources/test_playlist.py

```diff
@@ -30,3 +30,26 @@ def test_get_fans(self, playlist):
         assert isinstance(first_fan, deezer.User)
         assert first_fan.name == "Fay22"
         assert len(fans) == 100
+
+    def test_add_tracks(self, client_token):
+        playlist = client_token.get_playlist(11015569602)
+        # Test that we can add one track.
+        result = playlist.add_tracks([79875054])
```

Comment:
I think that it would be nice if this also worked by passing a `list`/`PaginatedResponse` of `Track`... What do you think?

(Same idea for the other methods below)

Reply:
Yes that would make sense, i can add it when I get home later

Reply:
Awesome! Thanks for the improvements 😊

Reply:
No problem, thanks for the library 😄